test.ts

```diff
@@ -1,10 +1,10 @@
-import Terraformer = require("./terraformer");
+import * as Terraformer from "./index";
 
 console.assert(typeof Terraformer !== undefined);
 
 let point = new Terraformer.Primitive({
   type: "Point",
-  coordinates: [1, 2]
+  coordinates: [1, 2],
```

Comment:
I've added a (TSLint)[http://palantir.github.io/tslint/] test to `npm run test:ts`. The tslint config has been modified to disallow the trailing commas. (By default it requires them. I have no preference either way on this matter.)